sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md

```diff
@@ -1,15 +1,10 @@
 # Release History
 
-## 3.0.0-beta.6 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
+## 3.0.0-beta.6 (2023-04-12)
 ### Other Changes
 
+- GA release
```

Comment:
I think this would be TODO, depends on the review and changes, but "GA release" is never a good changelog.

Reply:
test coverage is too low for GA
https://github.com/Azure/azure-sdk-for-java/blob/58280b0ddf826dbfeda9a637cf939791da02d71f/sdk/anomalydetector/azure-ai-anomalydetector/pom.xml#L41-L42

please check if you can cover all REST API endpoints.

also you may need better samples as well.